app/components/ak-tabs/item/index.hbs

```diff
@@ -0,0 +1,36 @@
+<AkList::Item
+  ...attributes
```

Comment:
this doesn't work so can remove this

Reply:
How do you mean? It works. A use case can be for testing attributes.